### DISCUSSION TITLE
Report - FTX24NMVJU not working

### DISCUSSION THREAD

**Discussion by turnrye** [revk/xESP32-Faikout#380] [General]
Hiya,

I just received a few faikins from amazon uk, thanks for making them so easy to get. I have a FTX24NMVJU that has a KRP980B2E installed to get an S21 port. I connected S21 with jumpers, ensuring that it was a straight through cable, and the faikin turns on, however the debug feature via mqtt shows the following:

debug:
```json
{"protocol":"S21¬Tx","cmd":"F1","noack":true}
```

dump:
```json
{"protocol":"S21¬Tx¬Rx","dump":"0246317703","F1":""}
```

Faikin-S3-MINI-N4-R2: 4cc882f 

I tried two different adaptor boards in the off chance that it was a bad board but got the same result.

**Comment by revk** [comment]
You can  lock settings to S21 only and not inverting tx and rx which helps avoid it cycling through options.

But if, even then, it just comes up with `noack`, that suggests something not connected right.

I'm assuming the topic on that second MQTT end `tx`

**Reply by turnrye** [reply]
Thanks for the tip -- I've enabled nox50a, nocnwired, noas, noswaptx, and noswaprx. Unfortunately it hasnt changed the result.

Here're my wiring hookup photos:
![56DE546D-A0FC-4A13-BF2A-01D4B433AA1C_1_102_o](https://github.com/revk/ESP32-Faikin/assets/701035/1d547be1-85a7-45ac-91a0-8e88b4e201b1)
![65D0E5F9-DAA2-46C3-9E9C-35949E00F74F_1_102_o](https://github.com/revk/ESP32-Faikin/assets/701035/028abb15-52d2-46a7-9789-b2666f2ac582)

info dump tx:
```json
{"protocol":"S21","dump":"0246317703","F1":""}
```
info dump rx:
```json
{"protocol":"S21","dump":"024731B17347616D03","G1":"\u00B1sGa"}
```

**Reply by revk** [reply]
If you are seeing an `rx` that suggests it should be working now! That response has too many bytes I think, which is odd.

**Reply by turnrye** [reply]
Unfortunately I think that was unrelated. Here's what's under the error comms topic still, with these happening on loop constantly:
```json
{"protocol":"S21","cmd":"F1","noack":true,"value":"66"}
{"protocol":"S21","cmd":"F1","noack":true,"value":"86"}
{"protocol":"S21","cmd":"F1","noack":true,"value":"27"}
{"protocol":"S21","cmd":"F1","noack":true,"value":"13"}
"{"protocol":"S21","timeout":true}"
```

Here are all of the RX dumps that have been captured, spanning the last 12 hours; this roughly lines up with the number of times we've changed the state using the IR remote; they dont seem to correlate with the debugs above:
```json
{"protocol":"S21","dump":"024731B17347616D03","G1":"\u00B1sGa"}
{"protocol":"S21","dump":"024731393347416403","G1":"93GA"}
{"protocol":"S21","dump":"026731713344416503","g1":"q3DA"}
{"protocol":"S21","dump":"024731713F64516103","G1":"q?dQ"}
{"protocol":"S21¬Rx","dump":"022903",")\u0003":"w\u0003)\u000F\u0082 \f\u0006"}
```

**Reply by revk** [reply]
OK there is some comms, but it is a mess. That is definitely an S21 port?

Anyone else on discussions doing the same?

Try with pin 1 not connected.

**Reply by turnrye** [reply]
I've also tried with pin 1 disconnected, unfortunately no change. Yes, this port is labeled S21 on the PCB and in the [submittal sheet](https://manuals.totalhomesupply.com/wp-content/uploads/manuals/KRP067A41E-KRP980B2E-Submittal.pdf) it's a mentioned feature.
![IMG_2792](https://github.com/revk/ESP32-Faikin/assets/701035/523492c1-e86a-4dd7-a9b2-83ac4a77a2b0)

![IMG_2791](https://github.com/revk/ESP32-Faikin/assets/701035/3f80a309-b19b-4825-888e-9f576a6c9645)

**Reply by revk** [reply]
OK I wonder if anyone else has used this board and can advise.

What voltage is on pin 1? (using pin 5 as GND)?

**Reply by turnrye** [reply]
Just measured +4.98VDC across pin 1 and 5. This also is consistent with the +5V that's printed on the board nearby (that via feeds pin 1 through a trace on the bottom of the board).

**Reply by revk** [reply]
OK that is good, so I am at a loss as to why this is not working.

Do you have a second Faikin to try?

**Reply by turnrye** [reply]
I do! I will try that. I also have a few more of these units in this series (various BTUs), so I can try different Daikins too :).

I will report back in about 12 hours with results.

**Reply by turnrye** [reply]
Actually I had another minute and plugged in a different faikin. This one worked immediately 🎊 . So at this point, we can treat the original post as resolved.

I'm still curious to understand _why_ this is happening. So later when I come back, I'll compare the configurations and settings side by side later to understand if this is a configuration issue or a hardware issue.

**Comment by turnrye** [comment]
Got a few more RX dumps since my original message:
```json
{"protocol":"S21","dump":"024771313348516503","Gq":"13HQ"}
{"protocol":"S21","dump":"02473171334E456D03","G1":"q3NE"}
{"protocol":"S21","dump":"0257393D3B78416503","W9":"=;xA"}
```

**Reply by revk** [reply]
Those look cleaner!

So maybe we are making progress.

Actually `Gq` and `W9` seem wrong, meaning it may be tx, so yeh, back to what voltage is on pin 1

**Comment by revk** [comment]
OK weird, return to amazon if faulty - could be a simple component fault.
